src/components/controls/measurementsOptions.tsx

```diff
@@ -32,15 +39,15 @@ const collectionOptionsSelector = (collections) => {
 
 const MeasurementsOptions = () => {
   const dispatch = useAppDispatch();
-  const collection = useSelector((state) => state.measurements.collectionToDisplay);
-  const collectionOptions = useSelector((state) => collectionOptionsSelector(state.measurements.collections), isEqual);
-  const groupBy = useSelector((state) => state.controls.measurementsGroupBy);
-  const display = useSelector((state) => state.controls.measurementsDisplay);
-  const showOverallMean = useSelector((state) => state.controls.measurementsShowOverallMean);
-  const showThreshold = useSelector((state) => state.controls.measurementsShowThreshold);
+  const collection = useSelector((state: RootState) => state.measurements.collectionToDisplay);
+  const collectionOptions = useSelector((state: RootState) =>  collectionOptionsSelector(state.measurements.collections), isEqual);
```

Comment:
<nit> extra whitespace

Reply:
Fixed up in https://github.com/nextstrain/auspice/pull/1910/commits/f5b3f2fb1d744abeade9383ed683219ef98b9cec